mmv1/products/contactcenterinsights/View.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: View
+description: |
+  Insights View resource for filtering conversations
+references:
+  api: 'https://cloud.google.com/contact-center/insights/docs/reference/rest/v1/projects.locations.views'
+base_url: projects/{{project}}/locations/{{location}}/views
+self_link: projects/{{project}}/locations/{{location}}/views/{{view}}
+create_url: projects/{{project}}/locations/{{location}}/views
+update_verb: PATCH
+update_mask: true
+id_format: projects/{{project}}/locations/{{location}}/views/{{view}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/views/{{view}}
+examples:
+  - name: 'contact_center_insights_view_basic'
+    primary_resource_id: 'basic_view'
+    vars:
+      resource_name: 'basic_view'
+      location_name: 'us-central1'
```

Comment:
The use of a variable for `location_name` automatically appends a random suffix to the variable value. There doesn't seem to be a need for location to be a variable in these examples, so you can hard code them to be `us-central1` in the yaml.